layers/meta-resin-raspberrypi/recipes-support/hostapp-update-hooks/files/99-rpi-bootloader

```diff
@@ -16,12 +16,11 @@ else
 fi
 
 new_part=$(findmnt --noheadings --canonicalize --output SOURCE $SYSROOT)
-blockdev=$(basename "$new_part")
-new_part_idx=$(cat "/sys/class/block/$blockdev/partition")
+new_part_label=$(blkid "$new_part" | awk '{print $2}')
 
-printf "[INFO] Switching RaspberryPi bootloader root partition index to %s..." "$new_part_idx..."
+printf "[INFO] Switching RaspberryPi bootloader root partition index to %s..." "$new_part_label..."
 cp /mnt/boot/cmdline.txt /mnt/boot/cmdline.txt.new
-sed -i "s#mmcblk0p.#mmcblk0p$new_part_idx#g" /mnt/boot/cmdline.txt.new
+sed -i "s/root=[^ ]* /"root=$new_part_label" /mnt/boot/cmdline.txt.new
```

Comment:
Please change the commit log to something like:

> hostapp-update-hooks: Change 99-rpi-bootloader to use filesystem labels for rootfs

And then in the rest of the commit log please give a more detailed description on why this change is needed. See https://github.com/resin-os/resin-raspberrypi/pull/152/commits/92bd8ade2b77dc9018ce6651bb9c0999ba939145 for reference

Also, please add a CHANGELOG.md entry with a short line for this change, like in: https://github.com/resin-os/resin-raspberrypi/pull/152/commits/0ab6e8df12dc2b01a0be52cbe26bc87a86e61eec

Reply:
CHangeLog.md done, but for the first change, do I have to do something or your change is append to teh initial commit?